src/steth/scUSDS.sol

```diff
@@ -20,32 +20,39 @@ contract scUSDS is ERC4626 {
     /// @notice The DAI ERC20 token contract.
     ERC20 public constant dai = ERC20(C.DAI);
 
+    /// @notice The sDAI ERC4626 token contract.
+    ERC4626 public constant sDai = ERC4626(C.SDAI);
+
     /// @notice The USDS ERC20 token contract.
     ERC20 public constant usds = ERC20(C.USDS);
 
     /// @notice The Dai - USDS converter contract from sky
     IDaiUsds public constant converter = IDaiUsds(C.DAI_USDS_CONVERTER);
 
     /// @notice The scDAI ERC4626 vault contract.
```

Comment:
change to scsDAI in the comment

Reply:
[Fixed](https://github.com/lindy-labs/sandclock-contracts/pull/166/commits/53b96260716a5b3cefb8eba44250546a7768c72f)